wordless/helpers/render_helper.php

```diff
@@ -18,38 +18,65 @@ function render_error($title, $description) {
     }
 
     /**
-    * Renders a template and its contained plartials. Accepts
-    * a list of locals variables which will be available inside
-    * the code of the template
-    *
-    * @param  string $name   The template filenames
-    *
-    * @param  array  $locals An associative array. Keys will be variables'
-    *                        names and values will be variable values inside
-    *                        the template
-    *
-    * @param boolean $static If `true` static rendering of PUG templates will
-    *                        be activated.
-    *
-    */
-    function render_template($name, $locals = array(), $static = false) {
+     * Retrive the actual template file path (searched in the views folder). It's possible to filter for an extension
+     * passing in the argument $force_extension_match the nane ex. "pug"
+     *
+     * @param  string $name                     The template filenames
+     * @param  array  $force_extension_match    The extension by which to filter the template file search
+     */
+    function template_info($name, $force_extension_match = '') {
```

Comment:
Since the present class is registered as a helper class by the ugly `Wordless::register_helper()` method, here we have the drawback to globalize this `template_info()` into Wordpress' global scope.

I'd ask you to test if:

* actually it's exported in the global scope (you could try it quickly from within `wp shell`)
* if it is, then declare the method as private and test if it is still working (it should from what I'm understanding)

TIA